Controllers/AdminController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Controllers;
+
+class AdminController  extends BasicController
+{
+
+  public function __construct()
+  {
+    parent::__construct();
+    $this->title = "Admin page";
+  }
+
+  /**
+   * Callback for /admin route.
+   */
+  public function adminPageAction() {
+    $this->content = $this->render('/views/layouts/admin_content.php');
```

Comment:
Please rename layout file to admin_content.tpl.php and move it under /views/admin folder.
We'll use /views/layouts folder for full pages templates.